src/config/index.js

```diff
@@ -61,39 +144,44 @@ class Config {
    */
   save() {
     // Delete unsaved scripts
-    fse
-      .readdir(SCRIPTS_DIRECTORY_PATH)
-      .then(scripts => {
-        scripts
-          .filter(scriptName => path.extname(scriptName) === '.yml')
-          .filter(scriptName => !this.scriptNames.includes(path.basename(scriptName, '.yml')))
-          .map(scriptName => `${SCRIPTS_DIRECTORY_PATH}/${scriptName}`)
-          .forEach(scriptName => deleteYamlFile(scriptName));
-      })
-      .catch(error => {
-        printMessage(
-          formatMessage(messages.errorDeletingScript, {
-            error,
-          }),
-        );
-      });
-
-    fse.outputJsonSync(
-      CONFIG_FILE_PATH,
-      {
-        shell: this.shell,
-        scriptNames: this.scriptNames,
-      },
-      {
-        spaces: 4,
-      },
-    );
+    let scripts;
+    try {
+      scripts = fse.readdirSync(SCRIPTS_DIRECTORY_PATH);
+    } catch (error) {
+      printMessage(
```

Comment:
nit:
- [ ] add a comment as to why we don't want to exit if there is an error deleting the script